doc/docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -270,4 +286,44 @@ Will create this schema
 
 |location|Point
 
-|===
\ No newline at end of file
+|===
+
+=== Read data by Relationship Type
+
+You can both specify a Cypher Path in this way:
+```scala
+val df = spark.read.format("org.neo4j.spark.DataSource")
+      .option("relationship", "BOUGHT")
+      .option("relationship.source.labels", "Person")
+      .option("relationship.target.labels", "Product")
+
+df.show()
+```
+
+This will create a Cypher Query as it follows:
+
+```cypher
+MATCH (source:Person)-[rel:BOUGHT]->(target:Product)
+RETURN source, rel, target
+```
+
+When reading data with this method, the Dataframe will contain all the fields contained in the relationship, plus:
+
+* `<id>` the internal Neo4j id
+* `<relatioshipType>` the relationship type
+
+and depending on the value of `relationship.node.map` option, if `true`:
+
+* `source` the Map<String, String> of source node
+* `target` the Map<String, String> of target node
+
+otherwise if `false`:
+
+* `<sourceId>` the internal Neo4j id of source node
+* `<sourceLabels>` a list of labels for source node
+* `<targetId>` the internal Neo4j id of target node
+* `<targetLabels>` a list of labels for target node
+
+==== Schema
+
+If APOC are installed, schema will be created with `apoc.meta.relTypeProperties`. Otherwise the first 10 (or any number specified by the `schema.flatten.limit` option) results will be flattened and the schema will be create from those properties.
```

Comment:
better to put this information with the option in the doc.  I missed it twice when reading through trying to figure what the. option did.